### PR TITLE
refactor: drop TuiSidebarModule

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/addon-mobile/directives/sidebar/sidebar.module.ts [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 import {TuiSidebarDirective} from './sidebar.directive';
 
 @NgModule({
-    imports: [TuiActiveZoneDirective, PolymorpheusModule],
-    declarations: [TuiSidebarDirective, TuiSidebarComponent],
+    imports: [TuiSidebarDirective, TuiSidebarComponent],
     exports: [TuiSidebarDirective],
 })
-export class TuiSidebarModule {}
+export class TuiSidebar {}
```
Comment (waterplea) [inline projects/addon-mobile/directives/sidebar/sidebar.module.ts:10]: There's no need for module at all, component is created dynamically, no need to import it.

File: projects/addon-doc/components/internal/header/header.module.ts [hunk 2]
```diff
@@ -13,7 +12,6 @@ import {TuiDocHeaderComponent} from './header.component';
         CommonModule,
         PolymorpheusModule,
         TuiButtonDirective,
-        TuiSidebarModule,
```
Comment (nsbarsukov) [inline projects/addon-doc/components/internal/header/header.module.ts:16]: Why ?
https://github.com/taiga-family/taiga-ui/blob/c55dfbf2f361c8a397fea6cce74edf22ff26ad83/projects/addon-doc/components/internal/header/header.template.html#L12

https://github.com/taiga-family/taiga-ui/assets/35179038/cddc4400-d460-48d5-b646-cc806c76f89b

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 3]
```diff
@@ -967,7 +967,7 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
             moduleSpecifier: '@taiga-ui/addon-mobile',
         },
         to: {
-            name: 'TuiTabBar',
+            name: 'TuiSidebarDirective',
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:970]: image

File: projects/addon-doc/components/internal/header/header.module.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-doc/components/internal/header/header.module.ts:16]: fixed

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts (hunk 3 above)
Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts]: fixed

**Review by mdlufy — APPROVED** [review]

**Pull request merged**